tests/ecosystem/deployment/test_deployment.py

```diff
@@ -30,9 +30,7 @@ def test_deployment(pvc_factory, pod_factory, bucket_factory, rgw_bucket_factory
             else:
                 sanity_helpers = Sanity()
             sanity_helpers.health_check()
-            sanity_helpers.create_resources(
-                pvc_factory, pod_factory, bucket_factory, rgw_bucket_factory
-            )
+            sanity_helpers.create_resources(pvc_factory, pod_factory)
```

Comment:
`create_resources()` expects `bucket_factory` and `rgw_bucket_factory` so I don't think this will work.
I suggest removing `sanity_helpers.create_resources()` from here as a temporary solution

Reply:
Looks like:
https://github.com/red-hat-storage/ocs-ci/blob/d949e68ee63435d916071fed7223ab528b0bf45c/ocs_ci/helpers/sanity_helpers.py#L48

Does it make sense to even run delete_resources if we will not run create_resources @ebenahar  ?

Reply:
@ebenahar i just reverted to which was before and i see deploy and destroy job was succeeded without issue. see logs here

https://github.com/red-hat-storage/ocs-ci/pull/3681

Reply:
curious how is the function signature mismatch working?

Reply:
> Looks like:
> https://github.com/red-hat-storage/ocs-ci/blob/d949e68ee63435d916071fed7223ab528b0bf45c/ocs_ci/helpers/sanity_helpers.py#L48
> 
> Does it make sense to even run delete_resources if we will not run create_resources @ebenahar ?

Since its initializing with empty list, delete resource won't cause any issues

Reply:
> curious how is the function signature mismatch working?

deleted the create_resource part as well

Reply:
> > curious how is the function signature mismatch working?
> 
> deleted the create_resource part as well

previous job skips the OCS installation, thats why it doesn't go through that part